firmware/inc/core1_sensor.h

```diff
@@ -0,0 +1,21 @@
+#ifndef CORE1_SENSOR_H
+#define CORE1_SENSOR_H
+
+#include <config.h>
+#include <pico/util/queue.h>
+#include "Adafruit_BME680_rpi.h"
+
```

Comment:
here. Referring to [this comment](https://github.com/AllenNeuralDynamics/harp.device.environment_sensor/pull/4/files#r1794198386).